main.nf

```diff
@@ -1120,10 +1181,14 @@ ch_dedup_for_pmdtools = Channel.empty()
 //Bamutils TrimBam Channel
 ch_for_bamutils = Channel.empty()
 
+//Population genetics Channel
+ch_for_pop_gen = Channel.empty()
```

Comment:
I did that before as well, but we should try to not use such setups. If its input, then add it to a channel. If its coming from another process, create the channel in that specific process.